frontend/package.json

```diff
@@ -44,7 +44,7 @@
         "rimraf": "^3.0.2",
         "style-loader": "^3.2.1",
         "webpack": "^5.47.1",
-        "webpack-bundle-tracker": "^1.2.0",
+        "webpack-bundle-tracker": "^1.4.0",
```

Comment:
added this change to sync versions of [django-webpack-loader](https://github.com/django-webpack/django-webpack-loader/blob/master/CHANGELOG.md).

> In order to use django-webpack-loader>=1.0.0, you must ensure that webpack-bundle-tracker@1.0.0 is being used on the JavaScript side. It's recommended that you always keep at least minor version parity across both packages, for full compatibility.

Not well described in the readme, but I think this makes sense...